client/version/self_update.go

```diff
@@ -48,16 +49,16 @@ func (su *SelfUpdater) Run() {
 }
 
 func (su *SelfUpdater) exit() {
-	go su.stack.Stop()
-
-	// wait 10 seconds for graceful shutdown
-	time.Sleep(time.Second * 10)
-
+	err := su.stack.Stop()
+	if err != nil {
+		su.logger.Error("Error stopping node", "err", err)
+	}
 	debug.Exit()
+	os.Exit(131)
 }
 
 func randomTo60SecDelay() {
-	randomTo60 := time.Duration(randomIntInRange(1, 60) * int(time.Second))
+	randomTo60 := time.Duration(randomIntInRange(1, 10) * int(time.Second))
```

Comment:
Isn't that 10 seconds?

Reply:
yeah, still working on it, just push so you can see if thats is a good way to restart the binary

Reply:
it seems like it will work, so sure, why not?  I'm not sure how to write a unit test for it, so I suppose we have to assume it will always do its job.